package.json

```diff
@@ -10,7 +10,8 @@
   "files": [
     "src",
```

Comment:
In order for the ignore rule to work, this one has to be more precise (like the other rules with extensions)